src/types/pathways-model.d.ts

```diff
@@ -60,6 +54,8 @@ declare module 'pathways-model' {
       cql: string;
       elm?: ElmLibrary;
     };
+    criteria?: string;
+    criteriaDisplay?: string;
```

Comment:
The Camino app right now just uses the `Transition.condition.description` for the display. Instead of creating a new property we should just populate that. It saves creating a new method in the builder interface and saves having to strip it when we export the pathway